core/src/main/java/nl/nn/adapterframework/util/MessageUtils.java

```diff
@@ -56,7 +56,10 @@ public static MessageContext getContext(HttpServletRequest request) {
 		result.withCharset(request.getCharacterEncoding());
 		int contentLength = request.getContentLength();
 		result.withSize(contentLength);
-		result.withMimeType(request.getContentType());
+		String contentType = request.getContentType();
+		if(StringUtils.isNotEmpty(contentType)) {
+			result.withMimeType(request.getContentType());
```

Comment:
Maybe this instead, to save some calls?
```suggestion
			result.withMimeType(contentType);
```

Reply:
I agree, you store the exact call above already so no need to call it again

Reply:
In this case it's a pointer to a field and not a value that needs to be calculated every time, but I like your way of thinking.